ta4j-core/src/main/java/org/ta4j/core/indicators/RandomWalkIndexHighIndicator.java

```diff
@@ -29,18 +29,16 @@ this software and associated documentation files (the "Software"), to deal in
 import org.ta4j.core.indicators.helpers.MinPriceIndicator;
 
 /**
- * The Class RandomWalkIndexHighIndicator.
+ * The Random Walk Index High Indicator. <p />
+ *
+ * See https://www.technicalindicators.net/indicators-technical-analysis/168-rwi-random-walk-index
  */
 public class RandomWalkIndexHighIndicator extends CachedIndicator<Decimal> {
```

Comment:
We should use the shorter filename "RWIHighIndicator.java" and "RWILowIndicator" instead of "RandomWalkIndexHighIndicator" and "RandomWalkIndexLowIndicator". Better to read and _"RWI"_ is the common abbreviation for **"RandomWalkIndex"**.